docker/manage

```diff
@@ -166,6 +166,7 @@ configureEnvironment() {
     CONTROLLER_WEB_HOOK_URL="${CONTROLLER_WEB_HOOK_URL}#${CONTROLLER_API_KEY}"
   fi
   export ST_ACAPY_ADMIN_API_KEY_NAME="x-api-key"
+  export CONTROLLER_CAMERA_REDIRECT_URL="https://id.gov.bc.ca/static/selfsetup.html"
```

Comment:
Good find! I'll check-in with the BC Wallet team to see if they intend to update this page, or will be creating their own.

Reply:
FYI: https://github.com/bcgov/bc-wallet-mobile/issues/989#issuecomment-1567081034